definitions/tools/cnvkit_vcf_export.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Convert default cnvkit .cns output to standard vcf format"
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: etal/cnvkit:0.9.5
+    - class: ShellCommandRequirement
+baseCommand: ["/usr/bin/python", "/usr/local/bin/cnvkit.py", "call"]
+arguments: [
+    { position: -1, valueFrom: $(inputs.male_reference), prefix: "-y" },
+    "-o", "adjusted.tumor.cns",
```

Comment:
Would it be useful to make this CNS file available as an output, too (even though this pipeline currently ignores it)?

Reply:
Cnvkit call only adds one additional field to the file, entirely based on numbers in the original file, and this data will be included in the vcf anyway, so I chose to ignore it. If for some reason it's needed, the original cns is currently an output, and this adjusted cns can be generated from it very quickly.